readthedocs/filetreediff/dataclasses.py

```diff
@@ -48,14 +55,105 @@ def as_dict(self) -> dict:
         return asdict(self)
 
 
-@dataclass
+class FileTreeDiffFileStatus(StrEnum):
+    """Status of a file in the file tree diff."""
+
+    added = auto()
+    deleted = auto()
+    modified = auto()
+
+    @property
+    def emoji(self) -> str:
+        """Return an emoji representing the file status."""
+        if self == FileTreeDiffFileStatus.added:
+            return "➕"
+        elif self == FileTreeDiffFileStatus.deleted:
+            return "❌"
+        elif self == FileTreeDiffFileStatus.modified:
+            return "📝"
+        return ""
+
+
+class FileTreeDiffFile:
```

Comment:
What's the difference between a `FileTreeDiffFile` and a `FileTreeDiffManifestFile`?

Reply:
One is used in the manifest (list of files and hashes), and the other one for the actual diff result.